src/component/NewComments.js

```diff
@@ -0,0 +1,45 @@
+import { postComment } from "../utils/api";
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+
+
+export const NewComment = ({  setGetComments, comments }) => {
+
+  const { article_id } = useParams();
+  console.log(article_id)
+  const [newComment, setNewComment] = useState("");
+
+
+  const handleSubmit = (e) => {
+
+    e.preventDefault();
+
+    setGetComments((comments)=>{
+
+      return [ {
+        article_id: article_id,
+        author: "grumpy19",
+        body: newComment,
+        comment_id: -1,
+        created_at: new Date().toISOString(),
+        votes: 0,
+      }, ...comments]
+    })
+      
+    postComment(article_id, newComment)
```

Comment:
I can currently post a blank comment, please could you add in logic to stop a user being allowed to do this